styles.css

```diff
@@ -0,0 +1,99 @@
+body {
+  margin: 0;
+}
+
+/* Navbar */
+.flex-Nav {
+  display: flex;
+  justify-content: space-between;
+  background: #3c3a39;
+  position: absolute;
+  padding: 0 5px 0 5px;
+  top: 0;
+  left: 0;
+  right: 0;
+}
+
+.nav-Button {
+  background-color: #3c3a39;
+  border: 0;
+  width: 14px;
+  flex-basis: 40px;
+  padding: 8px;
+  border-radius: 8px;
+}
+
+.nav-Button:hover {
+  background-color: black;
+}
+
+.header-Name {
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  line-height: 24px;
+  color: #fff;
+  padding: 10px 12px;
+  margin: 0;
+}
+
+/* headline section */
+#headline {
```

Comment:
- [ ] This whole section has a different color in the Figma design, to match it better, you should consider adding it to the style. This is the color used:
![image](https://user-images.githubusercontent.com/66967274/143365818-775c0949-5806-49bb-9704-533c336ea072.png)